client/i18n/locales/english/intro.json

```
@@ -95,16 +113,44 @@
           "In this course, you will use CSS to create your own painting in the style of Picasso. You will learn about FontAwesome SVG icons, CSS positioning, and reinforce the skills you have already been learning."
         ]
       },
-      "learn-responsive-web-design-by-building-a-piano": { "title": "Learn Responsive Web Design by Building a Piano", "intro": ["", ""] },
-      "learn-css-flexbox-by-building-a-photo-gallery": { "title": "Learn CSS Flexbox by Building a Photo Gallery", "intro": ["", ""] },
-      "learn-css-grid-by-building-a-magazine": { "title": "Learn CSS Grid by Building a Magazine", "intro": ["", ""] },
+      "learn-responsive-web-design-by-building-a-piano": {
+        "title": "Learn Responsive Web Design by Building a Piano",
+        "intro": [
+          "",
+          ""
+        ]
+      },
+      "learn-css-flexbox-by-building-a-photo-gallery": {
+        "title": "Learn CSS Flexbox by Building a Photo Gallery",
+        "intro": [
+          "",
+          ""
+        ]
+      },
+      "learn-css-grid-by-building-a-magazine": {
+        "title": "Learn CSS Grid by Building a Magazine",
+        "intro": [
+          "",
+          ""
+        ]
+      },
       "learn-typography-by-building-a-nutrition-label": {
         "title": "Learn Typography by Building a Nutrition Label",
-        "intro": ["", ""]
+        "intro": [
+          "",
+          ""
+        ]
       },
       "learn-css-transforms-by-building-a-penguin": { "title": "Learn CSS Transforms by Building a Penguin", "intro": ["", ""] },
       "learn-css-animation-by-building-a-ferris-wheel": { "title": "Learn CSS Animation by Building a Ferris Wheel", "intro": ["", ""] },
-      "learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet": { "title": "Learn More About CSS Pseudo Selectors By Building A Balance Sheet", "intro": ["", ""] }
+      "learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet": { "title": "Learn More About CSS Pseudo Selectors By Building A Balance Sheet", "intro": ["", ""] },
+      "learn-css-colors-by-building-a-colored-markers-set": {
+        "title": "Learn CSS Colors by Building a Colored Markers Set",
```

```suggestion
      "learn-css-colors-by-building-a-color-markers-set": {
        "title": "Learn CSS Colors by Building a Color Markers Set",
```
This crashes the page.

---

Oh man, thanks for catching that. I've had a bunch of trouble navigating to the challenges locally, and this is definitely why.